packages/sonarwhal/src/lib/utils/content-type.ts

```diff
@@ -382,11 +382,38 @@ const isTextMediaType = (mediaType: string): boolean => {
     return false;
 };
 
+const getType = (mediaType: string) => {
+    if (mediaType.startsWith('image')) {
```

Comment:
Do we want something special for the `favicon`?

Reply:
AFAIK no.
The getType method is for now the simplest method to get types, but we can improve it when we need more things :)